mcs/class/System.Windows.Forms/System.Windows.Forms/DataGridViewTextBoxCell.cs

```diff
@@ -211,14 +211,19 @@ protected override void Paint (Graphics graphics, Rectangle clipBounds, Rectangl
 				flags |= AlignmentToFlags (cellStyle.Alignment);
 
 				Rectangle contentbounds = cellBounds;
-				
-				contentbounds.Height -= 2;
-				contentbounds.Width -= 2;
-				
-				// If we are top aligned, give ourselves some padding from the top
-				if (((int)cellStyle.Alignment & 7) > 0) {
-					contentbounds.Offset (0, 2);
-					contentbounds.Height -= 2;
```

Comment:
can you explain why we don't need this anymore in the new code?

Reply:
@akoeplinger it looks like that code was a workaround to pad/offset with some "good enough" values when not processing advancedBorderStyle and cellStyle.Padding

Reply:
ideally this reasoning should go in the commit message, not in the PR